File: WebAPI/WebAPI/Entities/Deposit.cs
// Данный класс представляет собой вклад
public class Deposit : Account
{
    private int Percentage; // Процент вклада

    // Свойства
    public int percentage
    {
        get { return Percentage; }
        set { Percentage = value; }
    }

    // Конструктор
    public Deposit(int depID, int clID, string curr, int sum, int perc) : base(depID, clID, curr, sum)
    {
        this.percentage = perc;
    }
}

File: Front-end/src/entities/Deposit.ts
class Deposit{
    id: number;
    clientID: number;
    currency: string;
    sum: number;
    percentage: number;
    
    constructor(ID: number, cID: number, curr: string, Sum: number, perc: number){
        this.id = ID;
        this.clientID = cID;
        this.currency = curr;
        this.sum = Sum;
        this.percentage = perc;
    }
}